inst/mappings/mapping_AgMIP.csv

```diff
@@ -0,0 +1,995 @@
+EXPO.AGR;1000 t dm;Trade|Exports|Livestock products;Mt DM/yr;1000;NULL;
+EXPO.AGR;1000 t dm;Trade|Exports|Secondary products;Mt DM/yr;1000;NULL;
+EXPO.CGR;1000 t dm;Trade|Exports|Crops|Cereals|Maize;Mt DM/yr;1000;NULL;
+EXPO.CGR;1000 t dm;Trade|Exports|Crops|Cereals|Tropical cereals;Mt DM/yr;1000;NULL;
+EXPO.CGR;1000 t dm;Trade|Exports|Secondary products|Brans;Mt DM/yr;1000;NULL;
+EXPO.CGR;1000 t dm;Trade|Exports|Secondary products|Distillers grains;Mt DM/yr;1000;NULL;
+EXPO.CRP;1000 t dm;Trade|Exports|Crops;Mt DM/yr;1000;NULL;
+EXPO.CRP;1000 t dm;Trade|Exports|Secondary products;Mt DM/yr;1000;NULL;
+EXPO.DRY;1000 t dm;Trade|Exports|Livestock products|Dairy;Mt DM/yr;1000;NULL;
+EXPO.FSH;1000 t dm;Trade|Exports|Fish;Mt DM/yr;1000;NULL;
+EXPO.LSP;1000 t dm;Trade|Exports|Livestock products;Mt DM/yr;1000;NULL;
+EXPO.NRM;1000 t dm;Trade|Exports|Livestock products|Eggs;Mt DM/yr;1000;NULL;
+EXPO.NRM;1000 t dm;Trade|Exports|Livestock products|Monogastric meat;Mt DM/yr;1000;NULL;
+EXPO.NRM;1000 t dm;Trade|Exports|Livestock products|Poultry meat;Mt DM/yr;1000;NULL;
+EXPO.OCR;1000 t dm;Trade|Exports|Crops|Other crops|Potatoes;Mt DM/yr;1000;NULL;
+EXPO.OCR;1000 t dm;Trade|Exports|Crops|Other crops|Pulses;Mt DM/yr;1000;NULL;
+EXPO.OCR;1000 t dm;Trade|Exports|Crops|Other crops|Tropical roots;Mt DM/yr;1000;NULL;
+EXPO.OSD;1000 t dm;Trade|Exports|Crops|Oil crops|Groundnuts;Mt DM/yr;1000;NULL;
+EXPO.OSD;1000 t dm;Trade|Exports|Crops|Oil crops|Other oil crops incl rapeseed;Mt DM/yr;1000;NULL;
+EXPO.OSD;1000 t dm;Trade|Exports|Crops|Oil crops|Soybean;Mt DM/yr;1000;NULL;
+EXPO.OSD;1000 t dm;Trade|Exports|Crops|Oil crops|Sunflower;Mt DM/yr;1000;NULL;
+EXPO.OSD;1000 t dm;Trade|Exports|Secondary products|Oilcakes;Mt DM/yr;1000;NULL;
+EXPO.OSD;1000 t dm;Trade|Exports|Secondary products|Oils;Mt DM/yr;1000;NULL;
+EXPO.PFB;1000 t dm;Trade|Exports|Crops|Oil crops|Cotton seed;Mt DM/yr;1000;NULL;
+EXPO.PFB;1000 t dm;Trade|Exports|Secondary products|Cotton lint;Mt DM/yr;1000;NULL;
+EXPO.RIC;1000 t dm;Trade|Exports|Crops|Cereals|Rice;Mt DM/yr;1000;NULL;
+EXPO.RUM;1000 t dm;Trade|Exports|Livestock products|Ruminant meat;Mt DM/yr;1000;NULL;
+EXPO.SGC;1000 t dm;Trade|Exports|Crops|Sugar crops;Mt DM/yr;1000;NULL;
+EXPO.SGC;1000 t dm;Trade|Exports|Secondary products|Ethanol;Mt DM/yr;1000;NULL;
+EXPO.SGC;1000 t dm;Trade|Exports|Secondary products|Microbial protein;Mt DM/yr;1000;NULL;
+EXPO.SGC;1000 t dm;Trade|Exports|Secondary products|Molasses;Mt DM/yr;1000;NULL;
+EXPO.SGC;1000 t dm;Trade|Exports|Secondary products|Sugar;Mt DM/yr;1000;NULL;
+EXPO.VFN;1000 t dm;Trade|Exports|Crops|Other crops|Fruits Vegetables Nuts;Mt DM/yr;1000;NULL;
+EXPO.WHT;1000 t dm;Trade|Exports|Crops|Cereals|Temperate cereals;Mt DM/yr;1000;NULL;
+EXPO.WHT;1000 t dm;Trade|Exports|Secondary products|Alcoholic beverages;Mt DM/yr;1000;NULL;
+EXPO.ECP;1000 t dm;Trade|Exports|Bioenergy crops;Mt DM/yr;1000;NULL;
+NSUP_mass.CRP;TgN/year;Resources|Nitrogen|Pollution|Surplus|Cropland;Mt Nr/yr;1;NULL;
```

Comment:
Why does the AgMIP template use `TgN/year` here, while it uses `yr` instead of `year` everywhere else. Maybe worth harmonizing?